### ISSUE TITLE
Unable to run using America/Ciudad_Juarez TZ

### ISSUE THREAD

**Issue by vcano5** [louislam/uptime-kuma#3420]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### Description

The error occurs because a new time zone was recently created in this time zone due to the elimination of daylight saving time in Mexico. As a result, new time zones were created for the cities on the border with the United States. I'm not sure if the issue is with Dayjs or the Docker image being used.

https://unicode-org.atlassian.net/browse/CLDR-16181

![image](https://github.com/louislam/uptime-kuma/assets/30887270/b20a86f2-c18d-4247-98f8-6c45b286389c)

### 👟 Reproduction steps

1 .I created the container with the following command:
"docker run -d --restart=always -p 3001:3001 -v uptime-kuma:/app/data --name uptime-kuma louislam/uptime-kuma:1"
2. After checking the logs, that error appears.

### 👀 Expected behavior

Dashboard should be displayed when accessing IP:3001 

### 😓 Actual Behavior

Container crash, application not running

### 🐻 Uptime-Kuma Version

> 1.18.5

### 💻 Operating System and Arch

Ubuntu 22.04.2

### 🌐 Browser

Google Chrome 114.0.5735.199

### 🐋 Docker Version

20.10.18

### 📝 Relevant log output

```shell
Trace: RangeError: Invalid time zone specified: America/Ciudad_Juarez
    at new DateTimeFormat (<anonymous>)
    at /app/server/modules/dayjs/plugin/timezone.js:31:34
    at a (/app/server/modules/dayjs/plugin/timezone.js:42:14)
    at u (/app/server/modules/dayjs/plugin/timezone.js:46:21)
    at Function.o.tz (/app/server/modules/dayjs/plugin/timezone.js:92:21)
    at Logger.log (/app/src/util.js:111:25)
    at Logger.debug (/app/src/util.js:165:14)
    at UptimeKumaServer.initAfterDatabaseReady (/app/server/uptime-kuma-server.js:103:13)
    at async /app/server/server.js:178:5
    at process.<anonymous> (/app/server/server.js:1825:13)
    at process.emit (node:events:513:28)
    at emit (node:internal/process/promises:140:20)
    at processPromiseRejections (node:internal/process/promises:274:27)
    at processTicksAndRejections (node:internal/process/task_queues:97:32)
If you keep encountering errors, please report to https://github.com/louislam/uptime-kuma/issues
/app/server/modules/dayjs/plugin/timezone.js:31
                return r || (r = new Intl.DateTimeFormat("en-US", {
                                 ^
RangeError: Invalid time zone specified: America/Ciudad_Juarez
    at new DateTimeFormat (<anonymous>)
    at /app/server/modules/dayjs/plugin/timezone.js:31:34
    at a (/app/server/modules/dayjs/plugin/timezone.js:42:14)
    at u (/app/server/modules/dayjs/plugin/timezone.js:46:21)
    at Function.o.tz (/app/server/modules/dayjs/plugin/timezone.js:92:21)
    at Logger.log (/app/src/util.js:111:25)
    at Logger.debug (/app/src/util.js:165:14)
    at Timeout._onTimeout (/app/server/settings.js:36:21)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
```

**Comment by CommanderStorm** [comment]
See https://stackoverflow.com/questions/74820972/chrome-intl-cant-handle-america-ciudad-juarez

**Comment by CommanderStorm** [comment]
What is interesting is that the following code works on my browser (linux-chrome-114):

```js
const date = new Date();
const fmt=new Intl.DateTimeFormat('en-US', { timeZone: 'America/Ciudad_Juarez' }).format();
console.log(fmt);
```

**Comment by vcano5** [comment]
> Could you try running this in your console?
Output: 7/13/2023


I believe the issue lies on the server side with the Docker image/Dayjs library since I do everything through Portainer, and my Chrome installation is not involved.
It runs perfectly in 1.18.5 release

**Comment by chakflying** [comment]
`America/Ciudad_Juarez` is added in IANA tzdb `2022g`([changelog](https://data.iana.org/time-zones/tzdb/NEWS)).

The latest version of node.js [v16.x](https://github.com/nodejs/node/tree/v16.x/deps/icu-small) only ships with `2022f`. 16 is close to EOL so an update is unlikely.

I tested node.js `18.16.0` which does contain `2022g` and can handle the new timezone. We just needed to update the base image, but that will take some time.

In the meantime, maybe we should add try-catch to `log()` such that a logging error won't crash the server.
